SPECS/kernel-mos/kernel-mos.spec

```diff
@@ -0,0 +1,388 @@
+BuildRequires:  sed
+Requires:       filesystem
+Requires:       kmod
+Requires(post): coreutils
+Requires(postun): coreutils
+ExclusiveArch:  x86_64
+# When updating the config files it is important to sanitize them.
+# Steps for updating a config file:
+#  1. Extract the linux sources into a folder
+#  2. Add the current config file to the folder
+#  3. Run `make menuconfig` to edit the file (Manually editing is not recommended)
+#  4. Save the config file
+#  5. Copy the config file back into the kernel spec folder
+#  6. Revert any undesired changes (GCC related changes, etc)
+#  8. Build the kernel package
+#  9. Apply the changes listed in the log file (if any) to the config file
+#  10. Verify the rest of the config file looks ok
+# If there are significant changes to the config file, disable the config check and build the
+# kernel rpm. The final config file is included in /boot in the rpm.
+
+%description
+The kernel package contains the Linux kernel.
+
+%package devel
+Summary:        Kernel Dev
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       gawk
+Requires:       python3
+Obsoletes:      linux-dev
+
+%description devel
+This package contains the Linux kernel dev files
+
+%package drivers-accessibility
+Summary:        Kernel accessibility modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-accessibility
+This package contains the Linux kernel accessibility support
+
+%package drivers-gpu
+Summary:        Kernel gpu modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-gpu
+This package contains the Linux kernel gpu support
+
+%package drivers-sound
+Summary:        Kernel Sound modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-sound
+This package contains the Linux kernel sound support
+
+%package docs
+Summary:        Kernel docs
+Group:          System Environment/Kernel
+Requires:       python3
+
+%description docs
+This package contains the Linux kernel doc files
+
+%package tools
+Summary:        This package contains the 'perf' performance analysis tools for Linux kernel
+Group:          System/Tools
+Requires:       %{name} = %{version}-%{release}
+Requires:       audit
+
+%description tools
+This package contains the 'perf' performance analysis tools for Linux kernel.
+
+%package -n     python3-perf
+Summary:        Python 3 extension for perf tools
+Requires:       python3
+
+%description -n python3-perf
+This package contains the Python 3 extension for the 'perf' performance analysis tools for Linux kernel.
+
+%package dtb
+Summary:        This package contains common device tree blobs (dtb)
+Group:          System Environment/Kernel
+
+%description dtb
+This package contains common device tree blobs (dtb)
+
+%package -n     bpftool
```

Comment:
`mariner-2-mos` tag will likely update at a different cadence than the main kernel source tags. Would it make sense to submodule this rather than have this `kernel-mos` spec also provide a `bpftool` rpm?